src/index.ts

```diff
@@ -209,4 +209,4 @@ const plugin: PluginImpl<Options> = (options = {}) => {
   } satisfies Plugin;
 };
 
-export default plugin;
+export { plugin as dts } ;
```

Comment:
you can keep both the named export and the default export I believe

Reply:
If the plugin is both a named export and a default export, which should be documented in the README.md?

Maybe a hybrid approach with a note like #275 to say "if you're getting this error you can do a restructured import"?

Reply:
I believe you can document the newer named export in the README. But having the default export is rather a backwards compatibility thing, so existing users are not forced to upgrade.

Reply:
Also, having a comment such as that is still a good idea :-)

Reply:
- supporting default export in eeb269a4500d4d3d4a1c416709dbd3809cd93741
- add a note in README in cb57b11c883601477db0b997991b0190968733fa